### PR TITLE
New EasyConfig for non-Python version of libxml2 2.9.3 with the foss-2015a toolchain

### PR DESCRIPTION
**Pull request by valtandor**

### CODE REVIEW

File: easybuild/easyconfigs/l/libxml2/libxml2-2.9.3-foss-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+toolchain = {'name': 'foss', 'version': '2015a'}
+toolchainopts = {'pic': True}
+
+source_urls = [
+    'http://xmlsoft.org/sources/',
+    'http://xmlsoft.org/sources/old/'
+]
+sources = [SOURCELOWER_TAR_GZ]
+
+configopts = 'CC="$CC" CXX="$CXX" --with-pic --without-python'
```
Comment (boegel) [inline easybuild/easyconfigs/l/libxml2/libxml2-2.9.3-foss-2015a.eb]: @valtandor: please include `--with-zlib=$EBROOTZLIB` in here at the end, see #2551 and #2555

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/l/libxml2/libxml2-2.9.3-foss-2015a.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/l/libxml2/libxml2-2.9.3-foss-2015a.eb]: @valtandor ping on fixing this remark?

**Comment by valtandor** [comment]
@boegel done.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/3c659da477d4b43ba79a4405d31b254f for a full test report.

**Comment by boegel** [comment]
Going in, thanks @valtandor!

**Pull request merged**